arithmetic_analysis/false_position.py

```diff
@@ -0,0 +1,57 @@
+"""
+Created on Sat Feb  8 19:25:41 2020
+
+@author: MatteoRaso
+"""
+
+
+def main(f, lower_bound: float, upper_bound: float, error: float):
+    """An implementation of the false position method used for root-finding.
+    The algorithm is extremely similar to the secant method in that it uses
+    the x-intercept of the line connecting f(lower_bound) and f(upper_bound) 
+    in order to approximate the root of the function. The primary difference is that
+    the false position method has a defined interval that the root is bounded
+    by. This makes this method slightly slower but guarantees convergence.
+    
+    INPUTS:
+    f: The function which who's root we want to find.
+    lower_bound: A value at which f(lower_bound) < 0
+    upper_bound: A value at which f(upper_bound) > 0
+    error: The maximum tolerated error
+    
+    OUTPUT:
+    The approximated root bounded by lower_bound and upper_bound
+    """
+    if f(lower_bound) > 0:
```

Comment:
For edge case like these, how about raising an exception

Reply:
I'm not sure if I understand what you mean. The program would still run without that check, but it wouldn't be guaranteed to converge.

Reply:
```python
if f(lower_bound) > 0:
    raise ValueError("f(lower_bound) must be less than zero")

Reply:
Thank you for the clarification.